sdk/core/azure-core/src/test/java/com/azure/core/models/CloudEventTests.java

```diff
@@ -252,6 +254,33 @@ public void serializeJsonData() throws IOException {
         }
     }
 
+    @Test
+    public void serializeJsonDataWithObjectMapper() throws IOException {
```

Comment:
It would be good to add tests for each of the `BinaryDataContent` implementations to see if they can be serialized correctly when using an external `ObjectMapper`.

Reply:
Will do